src/System.Private.CoreLib/shared/System/IO/StreamWriter.cs

```diff
@@ -502,7 +503,7 @@ private void WriteFormatHelper(string format, ParamsArray args, bool appendNewLi
             StringBuilderCache.Release(sb);
         }
 
-        public override void Write(string format, object arg0)
+        public override void Write(string format, object? arg0)
```

Comment:
All these `Write` APIs are missing argument validation for `format` and the would null ref... should we fix them on a separate PR or at least open an issue?

Reply:
I fixed it in WriteFormatHelper.